packages/material-ui/src/Modal/Modal.js

```diff
@@ -393,6 +398,10 @@ Modal.propTypes = {
    * If `true`, hitting escape will not fire any callback.
    */
   disableEscapeKeyDown: PropTypes.bool,
+  /**
+   * If `true`, do not apply mui-fixed to the `Modal`
+   */
+  disableMuiFixed: PropTypes.bool,
```

Comment:
I agree with the name change but I suspect it will be neigh impossible fixing the problem without a similar property due to the Dialog being built on the Modal. Unless we change that

Reply:
Can Dialog not simply override the unwanted style? (Sorry if I"m missing something obvious here.)

Reply:
`mui-fixed` is just a marker and doesn't apply any styles directly. The modal manager applies paddingRight to every` mui-fixed` element. You can override the style but you will have to use the `!important` tag

Reply:
I was thinking of making it a hidden property initially not sure why I didn't